paradise_papers_search/fetch_api/models/Officer.py

```diff
@@ -12,4 +12,30 @@ class Officer(DjangoNode):
     addresses     = RelationshipTo('.Address.Address', 'REGISTERED_ADDRESS')
     entities      = RelationshipTo('.Entity.Entity', 'OFFICER_OF')
 
+    @property
+    def serialize(self):
+        return {
+            'node_properties': {
+                'sourceID': self.sourceID,
+                'name': self.name,
+                'country_codes': self.country_codes,
+                'valid_until': self.valid_until,
+                'countries': self.countries,
+                'node_id': self.node_id,
+            },
+        }
+
+    @property
+    def serialize_connections(self):
+        return [
+            {
+                'nodes_type': 'address',
+                'nodes_related': [address.serialize for address in self.addresses.all()],
+            },
+            {
+                'nodes_type': 'entity',
+                'nodes_related': [entity.serialize for entity in self.entities.all()],
+            },
```

Comment:
note: still need intermediaries  and  others  relationships. and a method like ` _entity_relationships` but for `Officer`.